package.js

```diff
@@ -2,6 +2,8 @@
 var packages = [
   'ecmascript',
   'promise',
+  'webapp',
+  'accounts-base',
```

Comment:
I would be careful with this one, I'd rather do a weak dependency, because accounts-base has a lot of deps around it. And it would be safer to just check for it's existence before doing accounts-related stuff